horace_core/sqw/@DnDBase/DnDBase.m

```diff
@@ -44,20 +44,22 @@
         obj = init_from_sqw_(obj, sqw_obj);
         obj = init_from_file_(obj, in_filename);
         obj = init_from_loader_struct_(obj, data_struct);
+
+        wout = sqw_eval_pix_(wout, sqwfunc, ave_pix, all_bins, pars);
```

Comment:
is this not calling a function that you thro an error in as an illegal call below? (Presuming that the lack of the all_bins ar does not make a difference)?

Reply:
Good catch, the API was changed and I forgot to update this signature